tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -179,8 +180,80 @@ def test_model_accuracy(model_name, input_shape):
         npt.assert_equal(expected.shape, actual.shape)
         npt.assert_almost_equal(expected, actual, decimal=3)
 
+def test_greater():
+    """Test for logical greater in onnx operators."""
+    input1 = np.array([[ 1,  1,  1], [ 1,  1,  1]]).astype("float32")
+    input2 = np.array([[ 0], [ 1]]).astype("float32")
+    op = np.array([[ 1,  1,  1], [ 0,  0,  0]]).astype("float32")
+    inputs = [helper.make_tensor_value_info("input1", TensorProto.FLOAT, shape=np.shape(input1)),
+              helper.make_tensor_value_info("input2", TensorProto.FLOAT, shape=np.shape(input2))]
+
+    outputs = [helper.make_tensor_value_info("output", TensorProto.FLOAT, shape=np.shape(op))]
+
+    nodes = [helper.make_node("Greater", ["input1", "input2"], ["output"])]
+
+    graph = helper.make_graph(nodes,
+                              "greater_test",
+                              inputs,
+                              outputs)
+
+    greater_model = helper.make_model(graph)
+
+    bkd_rep = backend.prepare(greater_model)
+    output = bkd_rep.run([input1, input2])
+    npt.assert_almost_equal(output[0], op)
+
+def test_equal():
+    """Test for equal in onnx operators."""
+    input1 = np.array([[ 1,  1,  1], [ 1,  1,  1]]).astype("float32")
+    input2 = np.array([[ 0], [ 1]]).astype("float32")
+    op = np.array([[ 0,  0,  0], [ 1,  1,  1]]).astype("float32")
```

Comment:
do not hard code this operation. use `np.equal`. Also can you please name this variable different, like `expected_value` or `numpy_op` or something like that.